extensions/src/ACRE2Arma/common/pbo/fileloader.hpp

```diff
@@ -1,22 +1,24 @@
 #pragma once
 
+#include "controller.hpp"
 #include "archive.hpp"
 #include "search.hpp"
 #include "singleton.hpp"
 #include "membuf.hpp"
-
+#include <shlwapi.h>
+#pragma comment(lib, "shlwapi.lib")
 
 namespace acre {
     namespace pbo {
         class file_entry {
         public:
-            file_entry() {};
-            file_entry(file_p entry_) : _file(entry_) { 
+            file_entry() = delete;            
```

Comment:
No need to delete. Simply delete the constructor.
```suggestion
            file_entry() = delete;            
```